src/include/sof/schedule/ll_schedule_domain.h

```diff
@@ -222,7 +222,8 @@ static inline bool domain_is_pending(struct ll_schedule_domain *domain,
 {
 	bool ret;
 
-	assert(domain->ops->domain_is_pending);
+	if (!domain->ops->domain_is_pending)
+		return true;
```

Comment:
you add a `.domain_is_pending` method to the DMA scheduling domain, so presumably this change isn't needed?